readme.md

```diff
@@ -78,131 +105,175 @@ For example:
 ```
 
 <details>
-<summary>When working with devise, what do you think might show up inside a flash message?</summary>
+  <summary><strong>When working with Devise, what do you think might show up inside a flash message?</strong></summary>
+
+  - Whether the user has signed in or out successfully
+  - Whether the password is correct/incorrect
+  - Whether the username is taken
 
-<ul>
-<li>Whether the user has signed in or out successfully</li>
-<li>Whether the password is correct/incorrect</li>
-<li>Whether the username is taken</li>
-</ul>
 </details>
 
 ### Commit!
 
 Seriously, commit now. It will be easy to fix any devise issues if you have a commit you can go back to.
 
-### Generate User model
+### You Do: Generate User Model (10 minutes / 0:35)
 
-```
+> 5 minutes exercise. 5 minutes review.
+
+In the User, Sessions & Flash lesson, you set up a User model manually. When using Devise, you should take advantage of its built in model generation functionality.
+
```

Comment:
Adjust the wording here. They didn't set up the the User model, it was already there in starter code.